noetic/distribution.yaml

```diff
@@ -2613,7 +2613,7 @@ repositories:
       tags:
         release: release/noetic/{package}/{version}
       url: https://github.com/ipa320/rqt_frame_editor_plugin-release.git
-      version: 1.1.0-1
+      version: 1.1.1-1
```

Comment:
@ptenbrock should the branch on both `source` and `doc` entries be `noetic-devel` instead?

Reply:
Indeed 1.1.1 appears to be in the noetic-devel branch instead of the kinetic-devel branch

Reply:
Thanks. You should be all set. Bloom pulled the correct version. We just want everyone else to get the correct version too.